pkgs/by-name/vr/vrcadvert/package.nix

```diff
@@ -3,25 +3,43 @@
   dotnetCorePackages,
   fetchFromGitHub,
   lib,
+  versionCheckHook,
 }:
 
 buildDotnetModule rec {
-  pname = "VrcAdvert";
+  pname = "vrcadvert";
   version = "1.0.0";
 
   src = fetchFromGitHub {
     owner = "galister";
     repo = "VrcAdvert";
-    rev = "v${version}";
+    rev = "refs/tags/v${version}";
     hash = "sha256-noIu5LV0yva94Kmdr39zb0kKXDaIrQ8DIplCj3aTIbQ=";
   };
 
-  dotnet-sdk = dotnetCorePackages.sdk_6_0;
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.sdk_8_0;
```

Comment:
Do we actually need to use the sdk as `dotnet-runtime` here?

Reply:
I misread it in the docs: `dotnet-runtime = dotnetCorePackage.sdk_8_0` :facepalm: